### DISCUSSION TITLE
haxeflixel.com and api.haxeflixel.com temporary mirrors

### DISCUSSION THREAD

**Discussion by jobf** [HaxeFlixel/flixel#2344] [General]
If anyone needs access to the documentation while haxeflixel.com is down there are now temporary versions here

https://haxeflixelcom-temp-mirror.netlify.app/

https://api-haxeflixelcom-temp-mirror.netlify.app/

**Comment by impaler** [comment]
its online again

**Reply by RichardBray** [reply]
Ah sweet thanks for getting it up @impaler. Guess we can use @jobf's URLs as a backup 🤷

**Reply by jobf** [reply]
I have put 301 redirects on the temp ones back to the proper sites.
